tests/common/helpers/dut_utils.py

```diff
@@ -177,7 +177,7 @@ def get_disabled_container_list(duthost):
     pytest_assert(succeeded, "Failed to get status ('enabled'|'disabled') of containers. Exiting...")
 
     for container_name, status in container_status.items():
-        if "disabled" in status:
+        if "disabled".encode('UTF-8') in status:
```

Comment:
I strongly recommend that we could transform those Unicode strings from the Ansible `command` or `shell` output dict into strings inside the `command` and `shell`, so users won't worry about encode/decode, @wangxin, what do you think?

Reply:
I think that's a good suggestion!